contrib/python/src/python/pants/contrib/python/checks/tasks/checkstyle/common.py

```diff
@@ -197,7 +198,7 @@ def iter_logical_lines(self, blob):
         yield self.translate_logical_line(
             line_number_start,
             token_start[0] + (1 if token_type is tokenize.NEWLINE else -1),
-            list(filter(None, contents)),
+            list([_f for _f in contents if _f]),
```

Comment:
I don't think the call to list is necessary, since this is a list comp

Reply:
correct